updatecli/updatecli.d/updatek3sroot.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: "Update k3sroot version" 
+
+sources:
+ k3sroot:
+   name: Get k3s-root version
+   kind: githubrelease
+   spec:
+     owner: k3s-io
+     repository: k3s-root
+     token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+     typefilter:
+       release: true
+       draft: false
+       prerelease: false
+     versionfilter:
+       kind: semver
+       # pattern accepts any semver constraint
+       pattern: "*"
+
+targets:
+  dockerfile:
+    name: "Bump to latest k3s-root version in Dockerfile"
+    kind: dockerfile
+    scmid: default
+    sourceid: k3sroot
+    spec:
+      file: "Dockerfile"
+      instruction:
+        keyword: "ARG"
+        matcher: "K3S_ROOT_VERSION"
+
+  makefile:
+    name: "Bump to latest k3s-root version in Makefile"
+    kind: file
+    scmid: default
+    disablesourceinput: true
+    spec:
+      file: Makefile
+      matchpattern: '(?m)^K3S_ROOT_VERSION \?\= (.*)'
+      replacepattern: 'K3S_ROOT_VERSION ?= {{ source "k3sroot" }}'
+
+scms:
+  default:
+    kind: github
+    spec:
+      token: '{{ requiredEnv "GITHUB_TOKEN"}}'
+      username: '{{ .github.username }}'
+      user: '{{ requiredEnv "GITHUB_ACTOR"}}'
+      email: '{{ .github.email }}'
+      owner: '{{ .github.owner }}'
+      repository: '{{ .github.repository }}'
+      branch: '{{ .github.branch }}'
+      
+actions:
+    default:
+        title: 'Bump K3s-root version to {{ source "k3sroot" }}'
+        kind: github/pullrequest
+        spec:
+            automerge: false
+            labels:
+                - chore
+                - skip-changelog
```

Comment:
Could we add here status/autogenerated?

Reply:
@rbrtbnfgl Do you man adding a label "status/auto-created" ?

Reply:
We are already using that label on the PR generated with the github actions.

Reply:
I did it already @rbrtbnfgl , could you review please? :)